### PR TITLE
39 newfocus tlb 670x driver erroneous response handling

### PR DESCRIPTION
**Pull request by heevasti**
Related to #39

Acceptance criteria 
=================== 
- [x] Functionality implemented in agreement with description of the ticket
- [x] Tests are updated
- [x] Documentation updated

### CODE REVIEW

File: qmi/instruments/newport/tlb670x.py [hunk 1]
```diff
@@ -151,7 +158,11 @@ def _receive(self) -> List[str]:
         return response
 
     def _get_device_info(self) -> List[Tuple[int, str]]:
-        """Get the contents of the device information buffer (a list of deviceID,deviceDescription pairs)."""
+        """Get the contents of the device information buffer (a list of deviceID,deviceDescription pairs).
+
+        Raises:
+            QMI_InstrumentException: In cases where getting the device info fails for some reason.
+        """
```
Comment (rbudhrani) [inline qmi/instruments/newport/tlb670x.py:165]: docstring missing returns

File: qmi/instruments/newport/tlb670x.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def __init__(self, context: QMI_Context, name: str, serial_number: Union[int, st
 
         # Prepare handle to DLL and device; set in open().
         self._handle: Any = ctypes.WinDLL(self.DLL_NAME)
-        self._device_id: int = 0
+        self._device_id: int = -1  # Initialize with non-existing device number. The real values are in range [0-31].
```
Comment (rbudhrani) [inline qmi/instruments/newport/tlb670x.py:70]: why this change?

**Review by rbudhrani — CHANGES_REQUESTED** [review]

File: qmi/instruments/newport/tlb670x.py (hunk 2 above)
Comment (heevasti) [reply qmi/instruments/newport/tlb670x.py:70]: For this: https://github.com/QuTech-Delft/QMI/pull/40/files#diff-2223e03ca4c943a12b3649a6bc0f13631a91ed2278392d664c9544e387d07469R209

File: qmi/instruments/newport/tlb670x.py (hunk 1 above)
Comment (heevasti) [reply qmi/instruments/newport/tlb670x.py:168]: Added Returns

**Review by rbudhrani — APPROVED** [review]
